browser_history/utils.py

```diff
@@ -4,6 +4,9 @@
 """
 import enum
 import platform
+import logging
+from browser_history import generic
+logging.basicConfig(level=logging.INFO)
```

Comment:
I think this sets the level for the root logger which would lead to info messages generated by other modules to also be shown. Having a global side-effect by importing a module does not seem like a good idea.

I would suggest defining a logger only for this module. Refer [this answer](https://stackoverflow.com/a/48787602/11199009)

Reply:
`import logging`
`logging.basicConfig()`
`logger=logging.getLogger(__name__)`
`logger.setLevel(logging.INFO)`
...
`logger.info("%s browser is not supported", browser_class.name)`


Would this be fine?

Reply:
Yes looks good, you can omit the `logging.basicConfig()` though.

Reply:
On  removing `logging.basicConfig()`, the issue faced is described [here](https://stackoverflow.com/questions/43109355/logging-setlevel-is-being-ignored)

Reply:
Right, I noticed that issue too, after trying it out. Though messing with the root logger is still not good.

I tried out another answer in that link, it works without messing with the root logger:

```suggestion
logger = logging.getLogger(__name__)
handler = logging.StreamHandler()
formatter = logging.Formatter('%(levelname)s: %(message)s')
handler.setFormatter(formatter)
logger.addHandler(handler)
logger.setLevel(logging.INFO)
```